stripe/src/main/java/com/stripe/android/paymentsheet/ui/PrimaryButtonAnimator.kt

```diff
@@ -63,6 +63,25 @@ internal class PrimaryButtonAnimator(
             -targetX
         ).also { animator ->
             animator.duration = slideAnimationDuration
+            animator.doOnEnd {
+                hold(view, parentWidth, onAnimationEnd)
+            }
+        }.start()
+    }
+
+    private fun hold(
```

Comment:
Nice! Maybe rename to `delayEnd`?

Reply:
how about delay, depending on where it is called it might be the end or somewhere else.

Reply:
sgtm